integration-tests/tests/advanced-happy-path.spec.ts

```diff
@@ -68,7 +68,11 @@ describe('Advanced Happy path', () => {
     });
 
     it('Add a component to Application', () => {
-      Applications.createComponent(repoLink, componentName, 'Go', true);
+      Applications.addComponentStep(repoLink);
+      Applications.configureComponentsStep(componentName, true, 'Go', {
```

Comment:
Should the `createComponent` method be extended to have optional env vars or is there some benefit of splitting it like this? I mean, if there already is a method `createComponent` which does both `addComponentSetup` and `configureComponentsStep`, why not just extend and reuse it?

Reply:
Internally `createComponent` component method calls both `addComponentStep` and `addComponentStep`
Here I just used both separately as there were more parameter to be passed in `createComponent`. 
In sort I just felt its more readable and understandable what its doing it just used in this way.
Although If you think `createComponent` with all the parameter is better option I can update that. do let me know.

Reply:
Updated